modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java

```diff
@@ -238,7 +238,10 @@
      * The feature allows authenticating a node remotely.
      * It is need when the one node which does not have an authentication required to authorize itself or another one.
      */
-    REMOTE_AUTH(61);
+    REMOTE_AUTH(61),
+
+    /** Snapshot operations with ZSTD, LZ4, SNAPPY algorithms. */
+    SNAPSHOT_COMPRESSION_MODERN_OPTION(63);
```

Comment:
I don't like naming here. Lets replace `MODERN` with `EXTENDED`, and when we need to add even more options, we'll add the same feature but with prefix V2 - see how it was done for SFTP_UPLOAD feature.